src/python/pants/backend/python/pex_util.py

```diff
@@ -22,13 +27,60 @@ def create_bare_interpreter(binary_path):
   return PythonInterpreter(binary_path, interpreter_with_extras.identity, extras=None)
 
 
-def get_local_platform():
-  """Returns the name of the local platform; eg: 'linux_x86_64' or 'macosx_10_8_x86_64'.
+def _interpreter_str(interp):
+  ident = interp.identity
+  return ('PythonInterpreter({binary!r}, {identity!r} with extended info: '
+          '(abbr_impl: {abbr_impl!r}, impl_ver: {impl_ver!r}, abi_tag: {abi_tag!r}))'
+          .format(binary=interp.binary,
+                  identity=ident,
+                  abbr_impl=ident.abbr_impl,
+                  impl_ver=ident.impl_ver,
+                  abi_tag=ident.abi_tag))
+
+
+def expand_and_maybe_adjust_platform(interpreter, platform):
```

Comment:
a TODO to upstream this into pex might be good?

Reply:
Yup - this old one https://github.com/pantsbuild/pex/issues/511, I'll add a TODO similar to the method above.

Reply:
Fixed.